frontend_build/src/read_bundle_plugins.js

```diff
@@ -12,15 +12,29 @@ var _ = require("lodash");
 
 var parseBundlePlugin = require('./parse_bundle_plugin');
 
+
+// Mappings for libraries that we bundle in the Kolibri core app.
+// * the keys are names exposed by webpack to use in `require` statements, across apps
+// * the values are references to the packages, already inserted into kolibriGlobal
+//
+// kolibri_name is always == kolibriGlobal (this is defined in the base settings - base.py)
+var libs = function(kolibri_name) {
```

Comment:
I am wondering if we can avoid defining this in two places (here and in the Kolibri constructor module) - will think on it.